src/server/images/models.js

```diff
@@ -94,8 +71,38 @@ export default class ImageManager {
   */
   static getImageObjList(imageKeys) {
     const s3 = new S3Connector();
-    return imageKeys.map((imageKey)=>{
+    return imageKeys.map((imageKey)=> {
       return {imageKey, imageUrl: s3.getImageUrl(imageKey)};
     });
   }
+
+ /**
+  * @getImageKeys
+  * get sorted image keys using item key.
+  * @param {string} itemKey imageKey
+  * @param {Array} checkState checkState
+  * @return {Array} image keys
+  */
+  static getImageKeys(itemKey, checkState = [STATE.ALIVE, STATE.EXPIRED]) {
+    const keys = [];
+    return Promise.all(checkState.map((state) => {
+      return new Promise((resolve, reject) => {
+        const prefix = KeyUtils.getPrefix(ENTITY.IMAGE, state, itemKey);
+        return fetchPrefix(prefix, (err, data) => {
+          if (err) {
+            return reject(err);
+          }
+          return resolve(data.map((value) => {
```

Comment:
I think you can remove variable `keys`

return resolve(data.map(value => value.key));